src/RawPOSIX/src/safeposix/dispatcher.rs

```diff
@@ -1148,19 +1148,27 @@ pub fn lindrustinit(verbosity: isize) {
     interface::cagetable_insert(0, utilcage);
     fdtables::init_empty_cage(0);
     // Set the first 3 fd to STDIN / STDOUT / STDERR
-    // STDIN
-    let dev_null = CString::new("/home/lind/lind_project/src/safeposix-rust/tmp/dev/null").unwrap();
+    // TODO:
```

Comment:
We can remove this todo note correct? Im pretty sure this is what you implemented here.